frontend/packages/ui/src/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { Toast, ToastProvider } from './Toast'
+
+const meta = {
+  component: Toast,
+  argTypes: {
+    status: {
+      control: 'select',
+      options: ['info', 'error', 'success', 'warning'],
+      description: 'The status of the toast',
+    },
+    isOpen: {
+      control: 'boolean',
+      description: 'Whether the toast is shown',
+    },
+  },
+  decorators: [
+    (Story) => (
+      <ToastProvider>
+        <Story />
+      </ToastProvider>
+    ),
+  ],
+} satisfies Meta<typeof Toast>
+
+export default meta
+type Story = StoryObj<typeof Toast>
+
+export const Info: Story = {
+  args: {
+    title: 'Info Toast',
+    description: 'Info Toast description',
+    status: 'info',
+    isOpen: true,
+  },
+}
+
+export const ErrorStatus: Story = {
```

Comment:
Name `Error` is not allowed by biome rule. https://biomejs.dev/linter/rules/no-shadow-restricted-names/